evm_arithmetization/src/cpu/kernel/asm/curve/bn254/curve_arithmetic/twisted_curve_checks.asm

```diff
@@ -92,3 +92,48 @@
     MUL // Cheaper than AND
     MUL // Cheaper than AND
 %endmacro
+
+%macro bn_check_twisted_subgroup
```

Comment:
nit: Maybe a comment pointing to https://eprint.iacr.org/2022/348 or #263 could be helpful to give some context here.